default/templates/default/snippets/optional_arguments.html

```diff
@@ -0,0 +1,9 @@
+{% load i18n %}
+<div class="form-group">
+	<label>{% trans "Encoding:" %}</label>
+	<input type="text" id="encoding" class="form-control" size="4" value="utf-8" >
```

Comment:
Isn't 4 too short? 'utf-8' has 5 characters

Reply:
I think that size attribute specifies only the visible width, it works with more than 4 characters but I increase it to 8 to make it look bigger.